pkg/core/faultinjections/matcher_test.go

```diff
@@ -131,6 +131,7 @@ var _ = Describe("Match", func() {
 				mesh_proto.InboundInterface{
 					WorkloadIP:   "1.2.3.4",
 					WorkloadPort: 8080,
+					InboundName:  "0",
```

Comment:
Shouldn't we fall back to `WorkloadPort` when there’s no explicit `InboundName`? Is that scenario even possible?

Reply:
shouldn't we also then adjust the tests so then InboundName is the same as WorkloadPort?

Reply:
I've updated tests to make it more accurate